plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/backup/TestFileBackupStore.java

```diff
@@ -18,37 +18,39 @@
 import org.testng.annotations.Test;
 
 import java.io.File;
+import java.io.IOException;
 import java.util.UUID;
 
-import static com.google.common.io.Files.createTempDir;
 import static com.google.common.io.MoreFiles.deleteRecursively;
 import static com.google.common.io.RecursiveDeleteOption.ALLOW_INSECURE;
 import static java.lang.String.format;
+import static java.nio.file.Files.createTempDirectory;
 import static org.testng.Assert.assertEquals;
 
 public class TestFileBackupStore
         extends AbstractTestBackupStore<FileBackupStore>
 {
     @BeforeClass
     public void setup()
+            throws IOException
     {
-        temporary = createTempDir();
-        store = new FileBackupStore(new File(temporary, "backup"));
+        temporary = createTempDirectory(null);
+        store = new FileBackupStore(temporary.resolve("backup").toFile());
         store.start();
     }
 
     @AfterClass(alwaysRun = true)
     public void tearDown()
             throws Exception
     {
-        deleteRecursively(temporary.toPath(), ALLOW_INSECURE);
+        deleteRecursively(temporary, ALLOW_INSECURE);
     }
 
     @Test
     public void testFilePaths()
     {
         UUID uuid = UUID.fromString("701e1a79-74f7-4f56-b438-b41e8e7d019d");
-        File expected = new File(temporary, format("backup/70/1e/%s.orc", uuid));
+        File expected = temporary.resolve("backup").resolve("70").resolve("1e").resolve(format("%s.orc", uuid)).toFile();
```

Comment:
Format as follows for clarity:
```suggestion
        File expected = temporary.resolve("backup")
                .resolve("70")
                .resolve("1e")
                .resolve(format("%s.orc", uuid))
                .toFile();
```

However, there's no need to chained calls. You can do:

```java
File expected = temporary.resolve(format("backup/70/1e/%s.orc", uuid));
```